app.py

```diff
@@ -21,6 +21,7 @@
 PORT_API_URL = config("PORT_API_URL", default="https://api.getport.io/v1")
 WEBHOOK_SECRET = config("WEBHOOK_SECRET", default="bitbucket_webhook_secret")
 IS_VERSION_8_7_OR_OLDER = config("IS_VERSION_8_7_OR_OLDER", default=False)
+GET_MERGED_PRS = config("GET_MERGED_PRS", default=False)
```

Comment:
When you pass ALL, it brings OPEN, MERGED and DECLINED prs. Based on that, can we rename the flag to  INGEST_ALL_PR_STATES or something equivalent?

Alternatively, you could make it PULL_REQUEST_STATE and allow them to pass ALL, OPEN, MERGED or DECLINED. then you set the default to OPEN.